typescript/README.md

```diff
@@ -0,0 +1,57 @@
+# TypeScript support for Winglang
+
+> TODO
+
+## Prerequisites
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+`sh
+npm i @winglibs/typescript
+`
+
+## Usage
+
+The `FunctionHandler`, `QueueHandler` and `ApiEndpointHandler` types can be used to define inflight
+handlers from a TypeScript codebase.
+
+For example:
+
+```ts
+// src/index.ts
+exports.handler = async (event, context) => {
+  console.log("Welcome to TypeScript", event);
+  return {
+    status: 200,
+    body: JSON.stringify({ message: "Hello, TypeScript" }),
+  };
+};
+```
+
+```js
+// main.w
+bring typescript;
+
+new cloud.Function(new typescript.FunctionHandler("./src"));
```

Comment:
(optional) I wonder if it's weird to recommend one lambda function per folder

```suggestion
new cloud.Function(new typescript.FunctionHandler("./src/index"));
```

Reply:
yes!

Reply:
>  I wonder if it's weird to recommend one lambda function per folder

@Chriscbr, How would this work for API gateway?

Reply:
done